<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VAA Ai-Go!</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabBar::tab{height:17}</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="LiveCamera">
       <attribute name="title">
        <string>Live</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="Processing">
       <attribute name="title">
        <string>Processing</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>617</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionContact"/>
    <addaction name="actionHelp_VAA_Ai_Go"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionSynchronizeStreams"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View(&amp;V)</string>
    </property>
    <addaction name="actionFullScreen"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuLanguages">
     <property name="title">
      <string>Languages</string>
     </property>
     <addaction name="actionEnglish"/>
     <addaction name="actionVietnamese"/>
    </widget>
    <widget class="QMenu" name="menuThemes">
     <property name="title">
      <string>Themes</string>
     </property>
     <addaction name="actionDark"/>
     <addaction name="actionWhite"/>
    </widget>
    <addaction name="menuLanguages"/>
    <addaction name="menuThemes"/>
    <addaction name="actionConfiguration"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuView"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSynchronizeStreams">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Synchronize streams</string>
   </property>
  </action>
  <action name="actionScaleToFitFrame">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Scale to fit frame</string>
   </property>
  </action>
  <action name="actionFullScreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Full Screen(&amp;F)</string>
   </property>
  </action>
  <action name="actionVietnamese">
   <property name="text">
    <string>Vietnamese</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionContact">
   <property name="text">
    <string>Contact</string>
   </property>
  </action>
  <action name="actionHelp_VAA_Ai_Go">
   <property name="text">
    <string>Help VAA Ai-Go!</string>
   </property>
  </action>
  <action name="actionConfiguration">
   <property name="text">
    <string>Configuration</string>
   </property>
  </action>
  <action name="actionDark">
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionWhite">
   <property name="text">
    <string>White</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
